Vyxal/Vyxal
# Mobile tooltips

closes #1115 #1116

to test:
- open it on your phone, or on desktop chrome with device toolbar and device set to an iphone or something. (firefox's touch simulation is broken.)
- tapping a key should type it.
- dragging through the keyboard should scroll.
- pressing and holding a key, and then dragging before 1000 ms passes, should cancel the dragging.
- pressing and holding a key, and then waiting for 1000 ms, should show the tooltip.
- pressing and holding a key, and then waiting for 1000 ms, and then dragging, should show different tooltips as you drag.
- pressing and holding a key, and possibly waiting or dragging, and then releasing, should type the last key touched (as long as it didn't scroll).

right now the threshold for showing the tooltip is "1000 ms", which in reality is absolutely not 1000 ms but whatever

unfortunately, to make this complex event handling work, i had to introduce preact
